nengo_viz/components/raster.py

```diff
@@ -10,6 +10,9 @@
 class Raster(Component):
     def __init__(self, viz, config, uid, obj, n_neurons=None):
         super(Raster, self).__init__(viz, config, uid)
+        if getattr(obj, 'neuron_type') is not None:
```

Comment:
I think it's impossible for `neuron_type` to be `None`, assuming that you can only make rasters for ensembles. So you should just be able to do `self.neuron_type = obj.neuron_type`. Then, below, rather than comparing strings, just check `if isinstance(self.neuron_type, nengo.Direct)`. In case we end up changing the string representation of `nengo.Direct` somewhere down the line!